pennylane_lightning/core/src/simulators/lightning_tensor/utils/cutn_utils/cutn_helpers.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2024 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file cutn_helpers.hpp
+ */
+
+#pragma once
+#include <cutensornet.h>
+#include <memory>
+#include <utility>
+
+#include "cutnError.hpp"
+
+namespace Pennylane::LightningTensor::Cutn::Util {
+
+enum MPSStatus {
+    MPSInitNotSet,
+    MPSInitSet,
+    MPSFinalizedNotSet,
+    MPSFinalizedSet
+};
```

Comment:
Should we prefer a scoped enumeration?
```suggestion
enum class MPSStatus : uint32_t {
    BEGIN = 0,
    MPSInitNotSet = 0,
    MPSInitSet,
    MPSFinalizedNotSet,
    MPSFinalizedSet,
    END
};
```

Reply:
sure, let's stick with C++ style.